app/scripts/modules/core/src/plugins/pluginSupport.ts

```diff
@@ -0,0 +1,38 @@
+import * as lodash from 'lodash';
+import * as react from 'react';
+import * as react_dom from 'react-dom';
+import * as spinnaker_core from '@spinnaker/core';
+
+export const pluginSupport = {
+  sharedLibraryNames: [
+    'lodash',
+    'react',
+    'react-dom',
```

Comment:
I see you have `@spinnaker/core` noted for removal/scoping down, but is it also fair to say `react-dom` will be unnecessary once we're past the experimental stage? Unless there's a great reason to have plugins calling `render()` and `createPortal()` on arbitrary DOM? I guess I assumed plugins would be calling explicit extension points that load them into the existing component tree.

Reply:
I was thinking `createPortal` is a legit use case

Reply:
yeah, out of the two `createPortal` feels more legitimate. I'm still somewhat skeptical though — my philosophy/approach with these sorts of things is usually to lock it down and wait to hear what kinds of use cases come up to see if there's a less loosey-goosey way of solving it. Either way I doubt we'll get ourselves into a particularly bad/troublesome situation and I don't have a strong preference. I'm somewhat biased from spending a long time in the world of extensions/third party code.